src/main/java/com/pnu/ordermanagementapp/view/ProductsExcelView.java

```diff
@@ -37,7 +37,7 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
             courseRow.createCell(idx++).setCellValue(product.getName());
             courseRow.createCell(idx++).setCellValue(product.getAmount());
             courseRow.createCell(idx++).setCellValue(product.getPrice());
-            courseRow.createCell(idx++).setCellValue(product.isActive());
+            courseRow.createCell(idx++).setCellValue(String.valueOf(product.isActive()));
```

Comment:
Can be better to have something like `product.isActive() ? "active" : "inactive"`